pipelines/snap-atac/snap-atac.options.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
Are we not using the options file for anything?

Reply:
I think in general we don't want to use options files for HCA pipelines since they run in terra, which doesn't support workflow options. Also the pipelines can run in DNANexus which (I think) also doesn't support options.

Reply:
👍